### ISSUE TITLE
Parquet (1662840)

### ISSUE THREAD

**Issue by ipr** [ValveSoftware/Proton#5255]
# Compatibility Report
- Name of the game with compatibility issues:Parquet
- Steam AppID of the game:1662840

## System Information
- GPU:  AMD Vega 10 XL/XT [Radeon RX Vega 56/64]
- Driver/LLVM version: Mesa 21.2.4
- Kernel version: 5.14.15
- Proton version:6.3

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
With Proton 6.3 there is black screen when starting game while you hear game sound playing. Clicking will get you to main menu which you can't see.

However, Proton 5.13-6 is working fine. So this is a regression in Proton 6.3.

There are other VNs with same thing: older Proton is working while newer does not. So there may be more games where Proton version is an issue for some reason.

## Reproduction
1. start game
2. observe black screen, listen to audio

[steam-1662840.log](https://github.com/ValveSoftware/Proton/files/7428676/steam-1662840.log)

**Comment by alasky17** [comment]
@ipr I tried to reproduce what you are seeing, but for me the game works on 6.3-7 in windowed mode but I get a black screen with music in fullscreen mode on both 5.13 and 6.3-7.  Could you verify that you are seeing the black screen in windowed mode on 6.3-7?

**Comment by ipr** [comment]
> @ipr I tried to reproduce what you are seeing, but for me the game works on 6.3-7 in windowed mode but I get a black screen with music in fullscreen mode on both 5.13 and 6.3-7. Could you verify that you are seeing the black screen in windowed mode on 6.3-7?

Switching to windowed mode leaves screen black in 5.13.
While starting there is a messagebox about D3D9 CreateDevice() failing.

**Comment by alasky17** [comment]
@ipr What about windowed mode on 6.3-7? Is that working?

**Comment by ipr** [comment]
> @ipr What about windowed mode on 6.3-7? Is that working?

Yes, after changing mode and starting game again it works in 6.3-7.

**Comment by alasky17** [comment]
@ipr Ok cool.  Just to confirm, you get a black screen on 6.3-7 in fullscreen mode, but the game works immediately in fullscreen mode on 5.13? I'm seeing a black screen in fullscreen mode on both Proton versions, and but I can "fix" it on both by hitting alt+f4 and then clicking "no" when it asks if I want to quit.

**Comment by ipr** [comment]
> @ipr Ok cool. Just to confirm, you get a black screen on 6.3-7 in fullscreen mode, but the game works immediately in fullscreen mode on 5.13? I'm seeing a black screen in fullscreen mode on both Proton versions, and but I can "fix" it on both by hitting alt+f4 and then clicking "no" when it asks if I want to quit.

Yes. I had shortly blackscreen on 5.13 as well but that is not repeating same way now for some reason. Whatever the cause is, it seems to be rather sensitive to something changing one way or another.

**Comment by alasky17** [comment]
@ipr Could you get a basic log of the game working in fullscreen mode on 5.13 and failing with the black screen on 6.3-7?  Just in case we can compare them and see what is going wrong?

**Comment by ipr** [comment]
> @ipr Could you get a basic log of the game working in fullscreen mode on 5.13 and failing with the black screen on 6.3-7? Just in case we can compare them and see what is going wrong?

Ok, here's logs. Let me know if there's something else that might be helpful.

[5.13-steam-1662840.log](https://github.com/ValveSoftware/Proton/files/7440792/5.13-steam-1662840.log)
[6.3-steam-1662840.log](https://github.com/ValveSoftware/Proton/files/7440794/6.3-steam-1662840.log)

**Comment by ipr** [comment]
Here's interesting difference.

In 5.13 there's this line:
warn:  winevulkan detected, disabling exclusive fullscreen support

That isn't in the log with 6.3.

**Comment by ipr** [comment]
With Proton experimental, kernel 5.16.8 and Mesa 21.3.5 there is black screen. Using PROTON_USE_WINED3D=1 makes display work again and game playable with this combination.

**Comment by alasky17** [comment]
@ipr I just retested this, and it looks like now this game works even without PROTON_USE_WINED3D=1?  I just tried Proton 7.0-5 and it is working now.  I have a feeling this was a dxvk bug fwiw, after reading back through this issue and the fact it worked with PROTON_USE_WINED3D for you.
